src/main/java/thePackmaster/util/AbstractOnAttackStance.java

```diff
@@ -0,0 +1,12 @@
+package thePackmaster.util;
+
+import com.megacrit.cardcrawl.actions.utility.UseCardAction;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.stances.AbstractStance;
+
+public abstract class AbstractOnAttackStance extends AbstractStance {
```

Comment:
Please make this an interface, not an abstract class. Also please move it into the stances package, not Util